canonical/global-nav
# 3.1.0

## Done

Updated the layout of the dropdown to include new branding according to this design: https://www.figma.com/file/BSP7FqnRMcMvO7kCozJKUa/Ubuntu.com-navigation-update-Iteration-9?node-id=4908%3A27365

## QA

- visit https://global-nav-225.demos.haus/
- see that the "All Canonical" dropdown content matches the design
